travis/spark.conf

```diff
@@ -1,14 +1,14 @@
 # Spark home
-hibench.spark.home      /opt/spark-1.6.0-bin-hadoop2.6/
+hibench.spark.home      /opt/spark-3.0.0-bin-hadoop2.7/
 
-# Spark version. Supported value: spark1.6, spark2.0, spark2.1, spark2.2
-hibench.spark.version   spark1.6
+# Spark version. Supported value: spark2.0, spark2.1, spark2.2, spark2.3, spark2.4, spark3.0
+hibench.spark.version   spark3.0
 
 # Spark master
 #   standalone mode: `spark://xxx:7077`
 #   YARN mode: `yarn-client`
 #   unset: fallback to `local[1]`
-hibench.spark.master    yarn-client
+hibench.spark.master    spark://localhost:7077
```

Comment:
Let's leave this unchanged, yarn may be an actually more popular choice.

Reply:
@gczsjdy  Thanks , I will modify it .

Reply:
@gczsjdy  Using yarn-client as master to spark-submit job , it will throw error (org.apache.spark.SparkException: Master must either be yarn or start with spark, mesos, k8s, or local) , because spark 3.0 has  deprecated using yarn-client and yarn-cluster .  So I  will set hibench.spark.master to  yarn.

`// Set the cluster manager on spark 3.0 
    val clusterManager: Int = args.master match {
      case "yarn" => YARN
      case m if m.startsWith("spark") => STANDALONE
      case m if m.startsWith("mesos") => MESOS
      case m if m.startsWith("k8s") => KUBERNETES
      case m if m.startsWith("local") => LOCAL
      case _ =>
        error("Master must either be yarn or start with spark, mesos, k8s, or local")
        -1
    }`